dadi/frontend/components/Table/Table.jsx

```diff
@@ -16,16 +16,89 @@ export default class Table extends Component {
     selectable: true
   }
 
+  constructor(props) {
+    super(props)
+
+    this.state.selectedRows = {}
```

Comment:
@eduardoboucas Aren't we using `static defaultProps` now? Why are we setting `state` outside of an action?